src/features/student/views/StudentUpdateAdditionalSkillView/components/AdditionalSkillCommentFormField/AdditionalSkillCommentFormField.test.ts

```diff
@@ -0,0 +1,153 @@
+import type { UpdateAdditionalSkillForm } from '@/features/student/views/StudentUpdateAdditionalSkillView/components/use-update-additional-skill-form/use-update-additional-skill-form'
+import type { VueWrapper } from '@vue/test-utils'
+import AdditionalSkillCommentFormField
+  from '@/features/student/views/StudentUpdateAdditionalSkillView/components/AdditionalSkillCommentFormField/AdditionalSkillCommentFormField.vue'
+import { BddTest } from '@avenirs-esr/avenirs-dsav/test-utils'
+import { useForm } from '@tanstack/vue-form'
+import { mountComponent } from 'tests/utils'
+import { beforeEach, expect, vi } from 'vitest'
+
+const TestWrapper = {
+  components: { AdditionalSkillCommentFormField },
+  setup () {
+    const form = useForm({
+      defaultValues: { comment: '' },
+      validators: {
+        onSubmit () {
+          return { fields: { comment: undefined } }
+        }
+      }
+    }) as unknown as UpdateAdditionalSkillForm
+
+    return { form }
+  },
+  template: `
+    <form @submit.prevent="form.handleSubmit">
+      <AdditionalSkillCommentFormField :form="form" />
+    </form>
+  `
+}
+
+BddTest().given('an additional skill comment form field component', () => {
+  let wrapper: VueWrapper
```

Comment:
```suggestion
  let wrapper: VueWrapper<InstanceType<typeof X>>
```

Il manque un typage sur VueWrapper